pkg/cache/sql/db/client.go

```diff
@@ -0,0 +1,312 @@
+/*
+Package db offers client struct and  functions to interact with database connection. It provides encrypting, decrypting,
+and a way to reset the database.
+*/
+package db
+
+import (
+	"bytes"
+	"context"
+	"database/sql"
+	"encoding/gob"
+	"os"
+	"reflect"
+	"sync"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/rancher/lasso/pkg/cache/sql/attachdriver"
+	"github.com/rancher/lasso/pkg/cache/sql/db/transaction"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"modernc.org/sqlite"
+	sqlite3 "modernc.org/sqlite/lib"
+)
+
+const (
+	// InformerObjectCacheDBPath is where SQLite's object database file will be stored relative to process running lasso
+	InformerObjectCacheDBPath = "informer_object_cache.db"
+	// OnDiskInformerIndexedFieldDBPath is where SQLite's indexed fields database file will be stored if, env var value
+	// found at EncryptAllEnvVar is "true".
```

Comment:
```suggestion
	// OnDiskInformerIndexedFieldDBPath is where SQLite's indexed fields database file will be stored if, env var value
	// found at EncryptAllEnvVar is "false".
```

My understanding is that if the CATTLE_ENCRYPT_CACHE_ALL env var is `true` then the index is in memory instead.